pinot-spi/src/main/java/org/apache/pinot/spi/utils/builder/ControllerRequestURLBuilder.java

```diff
@@ -194,7 +194,7 @@ public String forTableRebalance(String tableName, String tableType, boolean dryR
   }
 
   public String forTableReload(String tableName, TableType tableType, boolean forceDownload) {
-    String query = String.format("reload?type=%s&forceDownload=%s", tableType.name(), forceDownload);
+    String query = String.format("reload?forceDownload=%s&type=%s", forceDownload, tableType.name());
```

Comment:
Please revert the unrelated changes. I think something is not properly merged

Reply:
thanks for pointing out. reverted.